app/src/androidTest/java/com/example/triptracker/screens/home/HomeTest.kt

```diff
@@ -0,0 +1,279 @@
+package com.example.triptracker.screens.home
+
+import android.util.Log
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Home
+import androidx.compose.ui.test.assertHasClickAction
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.assertTextEquals
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithContentDescription
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import androidx.lifecycle.MutableLiveData
+import androidx.test.espresso.action.ViewActions.pressKey
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.example.triptracker.model.itinerary.Itinerary
+import com.example.triptracker.model.location.Location
+import com.example.triptracker.model.location.Pin
+import com.example.triptracker.model.repository.ItineraryRepository
+import com.example.triptracker.view.Navigation
+import com.example.triptracker.view.Route
+import com.example.triptracker.view.TopLevelDestination
+import com.example.triptracker.view.home.HomeScreen
+import com.example.triptracker.viewmodel.HomeViewModel
+import com.google.android.gms.maps.model.LatLng
+import io.github.kakaocup.compose.node.element.ComposeScreen
+import io.mockk.every
+import io.mockk.impl.annotations.RelaxedMockK
+import io.mockk.junit4.MockKRule
+import io.mockk.mockk
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class HomeTest {
+  @get:Rule val composeTestRule = createComposeRule()
+
+  // This rule automatic initializes lateinit properties with @MockK, @RelaxedMockK, etc.
+  @get:Rule val mockkRule = MockKRule(this)
+
+  // Relaxed mocks methods have a default implementation returning values
+
+  @RelaxedMockK lateinit var mockNav: Navigation
+  @RelaxedMockK private lateinit var mockViewModel: HomeViewModel
+  @RelaxedMockK private lateinit var mockItineraryRepository: ItineraryRepository
+
+  val mockItineraries =
```

Comment:
Maybe it would be useful to have a data class containing the template itineraries.
There are some also in the ItineraryListTest class.

Reply:
I agree, I will create a separate class with this generic attribute

Reply:
Done in the last commit, please let me know if that's what you wanted!